kbi/0013/index.rst

```diff
@@ -0,0 +1,95 @@
+.. index::
+   single: filter-branch; merge; copy-file
+
+.. highlight:: console
+
+KBI0013: Merge Data Availability Information
+============================================
```

Comment:
I feel like the title should somehow reflect the context of the text below (splitting a dataset). Right now it is really broad. I am not sure would would be best. Two possible directions

- Split a dataset and reuse data hosting
- Import content availability information on shared annex keys from another dataset

Reply:
I agree that "Merge Data Availability Information" was broad.

I'm still not crazy about the current title, but I wasn't able to come up with anything better.